extensions/wikia/Recirculation/js/plista.js

```diff
@@ -0,0 +1,70 @@
+define('ext.wikia.recirculation.plista', ['jquery'], function ($) {
+	function shouldFetchPlista(items) {
+		var index = items.findIndex(function (item) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'index' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "index" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)